packages/wonder-blocks-data/util/response-cache.js

```diff
@@ -125,19 +125,20 @@ export class ResponseCache {
         }
 
         const key = handler.getKey(options);
-        if (handler.invalidateCache(options)) {
+        const entry = handlerCache[key];
+        if (handler.invalidateCache(options, entry)) {
             delete handlerCache[key];
+            return null;
         }
```

Comment:
This needs to be changed. We still need the cache entry for the initial error state to be retrieved so that rehydration matches SSR, but we also need a new request to occur. I'll be making changes accordingly.